server/backend/cafeteria_rating_db_initializer.go

```diff
@@ -36,11 +36,11 @@ Writes all available tags from the json file into tables in order to make them e
 Will be executed once while the server is started.
 */
 func initTagRatingOptions(db *gorm.DB) {
-	updateTagTable("static_data/dishRatingTags.json", db, DISH)
+	/*updateTagTable("static_data/dishRatingTags.json", db, DISH)
```

Comment:
nit: Let's not cause this merge conflct ^^

```suggestion
	updateTagTable("static_data/dishRatingTags.json", db, DISH)
```